subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc

```diff
@@ -24,10 +24,10 @@ When configuring Gradle behavior you can use these methods, listed in order of h
 
 * <<command_line_interface.adoc#command_line_interface, Command-line flags>> such as `--build-cache`. These have precedence over properties and environment variables.
 * <<#sec:gradle_system_properties, System properties>> such as `systemProp.http.proxyHost=somehost.org` stored in a `gradle.properties` file.
```

Comment:
Not from your changes, but the first two times we mention this convention, we omit the fact that it only works for the root project

Reply:
https://github.com/gradle/gradle/pull/21006/files#diff-bf47e49178bd8fc0fbf491777da11ff52fbb8b8120da0368064b71fad6cc371cR194 may be the best place to male that clear.

Reply:
Good point, thanks! And fixed